content/_data/changelogs/weekly.yml

```diff
@@ -298,6 +298,7 @@
     - type: major rfe
       message: >
         <strong>Jenkins now requires Java 8 to run.</strong>
+         Note this applies to "slaves" too (you can configure a custom <code>JavaPath</code> in their "Advanced" properties)!
```

Comment:
I would rather say "It applies to all Jenkins components including master, agents, CLI and related components like Maven Interceptors in Maven Project Plugin"